cmd/flow-dps-indexer/main.go

```diff
@@ -68,6 +71,7 @@ func run() int {
 	pflag.BoolVarP(&flagForce, "force", "f", false, "force indexing to bootstrap from root checkpoint and overwrite existing index")
 	pflag.StringVarP(&flagIndex, "index", "i", "index", "path to database directory for state index")
 	pflag.StringVarP(&flagLevel, "level", "l", "info", "log output level")
+	pflag.StringVarP(&flagMetrics, "metrics", "m", "", "URL on which to expose metrics (no metrics are exposed when left empty)")
```

Comment:
We only need this for the live indexer, for operational purposes. There is no need to have it for past spork indexing, which should remain a command line utility (not serve something over a network).

Reply:
Past spork indexing can take an awful long time to complete, I think having some metrics there can always be useful, don't you think? It can also help to track down memory leaks, which could be an issue down the line due to the huge amount of operations it executes when indexing.

It doesn't cost much to have it, so why not? 🤷

Reply:
Because it's a command line tool.

Reply:
But one that might run for days at a time until it's done, no?

I'll remove it, but I still feel like it shouldn't be mutually exclusive :p

Reply:
The right way to track progress of the command line tool execution is log output, not metrics. Metrics are used for operations, for applications that _operate_ on an ongoing basis.